doc/DeveloperGuide.md

```diff
@@ -87,6 +87,27 @@ Use case ends.
 
 > 3a1. AddressBook shows an error message <br>
   Use case resumes at step 2
+  
+#### Use case: Rename an existing tag
+
+**MSS**
+
+1. User requests to list all tags
+2. AddressBook shows a list of tags
+3. User requests to rename a specific tag in the list
+4. AddressBook renames the existing tag
```

Comment:
As stated in the LO description, AddressBook should ask for confirmation from user before proceeding at this step.